doc/version_history.rst

```diff
@@ -6,6 +6,11 @@
 Version History
 ###############
 
+v1.20.1
+-------
+
+* Improve the `rules.MTForceError` rule.
```

Comment:
v1.20.0 has not been released yet. Please add this to that version.

Reply:
Updated in the commit: [4c76002](https://github.com/lsst-ts/ts_watcher/pull/87/commits/4c76002fd1f2bc6a963e833840596a65117225c8). Thanks!